blog/2020-04-02-march-dev-update.md

```diff
@@ -0,0 +1,46 @@
+---
+title:  "March, 2020 - Dev Update"
+author: daniel
+author_url: https://github.com/da-kami
+author_image_url: https://avatars1.githubusercontent.com/u/5557790
+tags: [dev-update]
+---
+
+> The sky is everywhere, it begins at your feet.
+>
+> — Jandy Nelson
+
+[Join us for the next COMIT Community ☕️](https://meet.jit.si/COMIT-Community-Coffee-04-2020) today, 7pm Sydney time, or listen in on [youtube](https://youtu.be/DHbGsf9i8TM)!
+
+Today's community coffee we will reflect on the first quarter of this year with a focus on Lightning Network (layer 2) integration into COMIT.
+We will discuss advantages and possible pitfalls. 
+Furthermore, we will discuss decentralised order-book solutions and what we want to tackle in the second quarter.
+
+<!--truncate-->
+
+We are moving fast.
+But sometimes things need time - the Lightning integration into the COMIT core is almost done... and will be finished shortly.
+In order to speed things up for integrating Lightning we now have a squad of three people working on that task only.
+With the focus squad you can be expected within the next week.
+Once the [Lightning integration epic](https://github.com/coblox/admin/issues/158) is closed it's done! Stay tuned.
+
+The [proof of concept for Grin-BTC swaps](https://github.com/comit-network/grin-btc-poc) was successfully finished by the end of March.
+We are not moving on to integrating the solution outlined in th proof of concept into the core of COMIT.
+We are already very excited on integrating privacy preserving swaps as new protocol powered by scriptless scripts.
+
+We started investing time using our own tools to improve them and learn from them.
+You can have a first glimpse of the results of a [BTC-DAI swap app prototype](https://github.com/comit-network/app).
+
+The documentation is still improving on our homepage.
+Additionally the homepage will soon get a new improved start-page. Stay tuned!
+
+Happy hacking,
+
+— Daniel
+
+If you want to take a closer look at what we did last month, here are the GitHub issues closed in January from our main repositories:
+
+- [comit-rs](https://github.com/comit-network/comit-rs/issues?utf8=%E2%9C%93&q=is%3Aissue+sort%3Aupdated-desc+closed%3A2020-03-01..2020-03-31)
+- [create-comit-app](https://github.com/comit-network/create-comit-app/issues?utf8=%E2%9C%93&q=is%3Aissue+sort%3Aupdated-desc+closed%3A2020-03-01..2020-03-31)
+- [comit-js-sdk](https://github.com/comit-network/comit-js-sdk/issues?utf8=%E2%9C%93&q=is%3Aissue+sort%3Aupdated-desc+closed%3A2020-03-01..2020-03-31)
+- [blockchain-contracts](https://github.com/comit-network/blockchain-contracts/issues?utf8=%E2%9C%93&q=is%3Aissue+sort%3Aupdated-desc+closed%3A2020-03-01..2020-03-31)
```

Comment:
Clicking on this last link is not all that impressive lol. Maybe we should leave it out?

Reply:
good point, I'll remove them :)